src/runtime_src/core/tools/xbutil2/OO_HostMem.cpp

```diff
@@ -107,14 +107,13 @@ OO_HostMem::execute(const SubCmdOptions& _options) const
     throw xrt_core::error(std::errc::operation_canceled);
   }
 
-  bool enable = false;
   try {
-    if ((boost::iequals(m_action, "ENABLE") != 0) && (boost::iequals(m_action, "DISABLE") != 0)) {
+    bool enable = boost::iequals(m_action, "ENABLE");
+    if (!enable && !boost::iequals(m_action, "DISABLE")) {
       std::cerr << boost::format("ERROR: Invalid action value: '%s'\n") % m_action;
       printHelp();
       throw xrt_core::error(std::errc::operation_canceled);
     }
-    enable =  boost::iequals(m_action, "ENABLE");
```

Comment:
And then execute this line of code:
`enable =  boost::iequals(m_action, "ENABLE");`
Since we know it will be the correct value from this point forward.

Reply:
Sounds good! I was not sure what was the correct choice to make here, thanks for the tip! :)